### ISSUE TITLE
The second technical seminar - 14:00 19 May 2019

### ISSUE THREAD

**Issue by itschunv** [vietnamese-engineer-in-japan/seminar#2]
**The second technical seminar on 19 May 2019**
- Content:
   + @sontdhust - Son Tran: Head first convolutional neural network
   + @yuyuvn   -- Duc Phan: Bluetooth LE

- Information: 
   + Location: Trung's apartment, meguro, tokyo, japan
   + Time: 14:00 19 May 2019

**Comment by itschunv** [comment]
**Rules:**
- Keep all devices in silent mode.
- Don't eat/drink on my bed, sitting/lying is ok. 
- Don't leave any trash on the floor. (I always have to clean my house after the seminar/workshop)	
- On time: starting time, researching time and discussing time
- Share all your knowledge
- Don't disturb presenters/discussion

**Comment by itschunv** [comment]
**Pre-preparation:**

- To be updated!

**Comment by itschunv** [comment]
## Materials

**Presenter**: Son Tran @sontdhust 
**Topic**: **Head first convolutional neural network**
**Slide**:
> https://metalwhale.github.io/cnn-slide/#slide=1

**Comment by itschunv** [comment]
**Presenter**: Duc Phan @yuyuvn 
**Topic**: **Bluetooth LE**
**Slide**:
> https://docs.google.com/presentation/d/1qnsI-W1BXotahIiTH9A40gyIihHU2m54eU64UUSyzlQ/edit#slide=id.p

**Comment by itschunv** [comment]
13 members, gather in Meguro (in image + Trung Vu)

**Shots:**
![seminar2nd](https://user-images.githubusercontent.com/17500705/57979539-1afdd980-7a5a-11e9-85c4-c93f75fe5e11.jpg)

**Comment by tl39** [comment]
looks better than the last time :D

**Comment by dungvt93** [comment]
> ## Materials
> **Presenter**: Son Tran @sontdhust
> **Topic**: **Head first convolutional neural network**
> **Slide**:
> 
> > https://metalwhale.github.io/cnn-slide/#slide=1

two question for son tran sama
+hyper param batch size effect to speed of trainning but is it effect to accuracy ?
+Don't see dropout layer in slide but dropout layer can be use with convolution layer?

**Comment by sontdhust** [comment]
Dear @dungvt93, let me answer your two questions:
> +hyper param batch size effect to speed of trainning but is it effect to accuracy ?

**TLDR**: AFAIK, Batch size has no effect to the accuracy, since it is not included in model structure.
[Reference](https://machinelearningmastery.com/difference-between-a-parameter-and-a-hyperparameter/)

> +Don't see dropout layer in slide but dropout layer can be use with convolution layer?

Basically, you can add dropout everywhere you want, it is up to you, even inside the convolutional layers. But you should not do this. Please refer to this [post](https://www.reddit.com/r/MachineLearning/comments/42nnpe/why_do_i_never_see_dropout_applied_in/) for more information.
